tdrs-backend/tdpservice/settings/common.py

```diff
@@ -300,6 +300,9 @@ class Common(Configuration):
             "rest_framework.authentication.SessionAuthentication",
             "rest_framework.authentication.TokenAuthentication",
         ),
+        "DEFAULT_FILTER_BACKENDS": [
+            "django_filters.rest_framework.DjangoFilterBackend",
```

Comment:
This package was already installed but previously unused due to this setting being missing and no view defining a filter backend.

https://www.django-rest-framework.org/api-guide/filtering/#setting-filter-backends